<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>823</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>等线</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>自动组卷评卷考试系统 By:Qingsong Liu</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>821</width>
     <height>571</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="tabletTracking">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgba(0,0,0,0.8);
border-radius:15px;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>80</y>
      <width>321</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:40px;
font-family:等线;

color:white;
font-size:40px;
background:transparent;</string>
    </property>
    <property name="text">
     <string>LOGIN HERE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>460</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:red;
color:white;
font-size:30px;
border-radius:15px;
}
QPushButton:hover{
background:#808080;
border-radius:15px;
}</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
    <property name="iconSize">
     <size>
      <width>5</width>
      <height>5</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:#717072;
font-size:36px;
font-family:等线;
border-bottom:3px solid #717072;
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>651</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;
color:#717072;
font-size:36px;
font-family:等线;
border-bottom:3px solid #717072;
border-radius:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton1">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font-size:28px;
font-family:微软雅黑;
color:white;</string>
    </property>
    <property name="text">
     <string>教师端</string>
    </property>
    <property name="iconSize">
     <size>
      <width>17</width>
      <height>8</height>
     </size>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>350</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
font-size:28px;
font-family:微软雅黑;
color:white;</string>
    </property>
    <property name="text">
     <string>学生端</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:red;
color:white;
font-size:40px;
font-family:微软雅黑;
border-top-right-radius:0px;
border-top-left-radius:0px;
border-bottom-right-radius:10px;
border-bottom-left-radius:10px;
}

QPushButton:hover{
background:#808080;
}</string>
    </property>
    <property name="text">
     <string>×</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_min">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>0</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:red;
color:white;
font-size:50px;
font-family:微软雅黑;
border-top-right-radius:0px;
border-top-left-radius:0px;
border-bottom-right-radius:10px;
border-bottom-left-radius:10px;
}

QPushButton:hover{
background:#808080;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="toolButton1">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>141</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:red;
border-radius:70px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../qrcimage.qrc">
     <normaloff>:/新前缀/image/user3.jpg</normaloff>:/新前缀/image/user3.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../qrcimage.qrc"/>
 </resources>
 <connections/>
</ui>
